src/jslog/jslog.h

```diff
@@ -11,7 +12,7 @@ enum JSLogLevel {
   JS_LOG_LEVEL_REL_INFO
 };
 
-#define JS_LOG_LEVEL_DEFAULT JS_LOG_LEVEL_INFO
+#define JS_LOG_LEVEL_DEFAULT JS_LOG_LEVEL_DEBUG
```

Comment:
i made JS_LOG_LEVEL_DEBUG as defalut value because otherwise i didn't found a solution to let debug messages to be dispalyed.